tools/o3de-doxygen/README.md

```diff
@@ -0,0 +1,184 @@
+
+
+1. Install Rust by using [rustup](https://rustup.rs/), or by running the following command in your Linux machine: 
+
+    ```shell
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+    ```
+
+1. Add rust to your environment variables.
+
+    ```shell
+    source "$HOME/.cargo/env"
+    ```
+
+1. Install the `fd-find` cargo, which is a file searching tool:
+
+    ```shell
+    cargo install fd-find
+    ```
+
+
+
+### Clean repos
+
+The C++ API reference generation scripts runs Doxygen on your local `o3de` clone. Then it outputs the generated files into your local `o3de.org` clone. Therefore, before you generate the API, you must ensure that your cloned repos are clean and have the latest changes.
+
+#### `o3de`
+
+1. In your local `o3de` clone, `git checkout` the latest updates in the `development` branch, or in the case of a major O3DE release, the `stabilization` branch. 
+
+    ```shell
+    git fetch --all
+    git checkout upstream/<branch> 
+    ```
+   -  **upstream**: Set to track the source repo, [`https://github.com/o3de/o3de.git`](https://github.com/o3de/o3de.git)
+
+1. Make sure the branch is clean from any untracked files or changes. The branch should be at the same state as the upstream branch's HEAD. To clean, run  `git clean`, `git reset`, or `git stash` as needed.
+
+#### `o3de.org`
+
+1. In your local `o3de.org` clone, `git checkout` the latest updates in the `development` branch, or in the case of a major O3DE release, the `stabilization` branch. 
+
+    ```shell
+    git fetch --all
+    git checkout upstream/<branch> 
+    ```
+   -  **upstream**: Set to track the source repo, [`https://github.com/o3de/o3de.org.git`](https://github.com/o3de/o3de.org.git)
+
+1. Create a new branch, such as `api-ref`. This branch will contain the API reference docs that you will [generate](#generate) later.
+
+    ```shell
+    git switch -c api-ref
+    ```
+
+1. Make sure the branch is clean from any untracked files or changes. The branch should be at the same state as the upstream branch's HEAD. To clean, run  `git clean`, `git reset`, or `git stash` as needed.
+
+### Prepare scripts
+
+1. In `config.sh` set the following variables:
+
+   - `O3DE_PATH`: Path to your local o3de clone  (e.g. $HOME/o3de)
+
+   - `O3DEORG_PATH`: Path to your local o3de.org clone (e.g. $HOME/o3de.org)
+
+    The C++ API reference generation scripts read from `o3de` and write to `o3de.org`. 
+
+1. If this is your first time running the script, you may need to enable permissions to execute the program. In the `o3de-doxygen` directory, run the following: 
+
+```shell
+cd <path-to>/o3de-doxygen
+chmod +x *.sh
+```
+
+## Generate
+
+The following scripts generate the frameworks and Gems APIs. 
+
+### Frameworks API
+
+In the `o3de-doxygen` directory, run the script:
+
+```shell
+./o3de-framework-apis.sh
+```
+
+This generates the following files in `o3de.org`: 
```

Comment:
Since you're referring to files in an output directory, I'd suggest using the path that you had them define.
```suggestion
This generates the following files in `O3DEORG_PATH`: 
```